README.md

```diff
@@ -15,75 +15,74 @@
 </p>
 <p>&nbsp;</p>
 
-Hops is a modular development, build and runtime environment for universal ("isomorphic") web applications. It focuses on [React](https://facebook.github.io/react/).
+Hops is everything you need to develop and deploy a production grade universal web application with [React](https://facebook.github.io/react/). It provides both a **universal runtime** as well as the necessary **build tooling**.
 
-Hops is designed to simplify getting started with modern frontend tooling. It streamlines [Webpack](https://webpack.js.org) and [Jest](https://facebook.github.io/jest/) configuration featuring [Babel](https://babeljs.io) and [PostCSS](http://postcss.org).
+### Philosophy
 
-### Quick start
+Hops targets the the professional React developer and follows React mainstream best practices
 
-If you just want to get started with hops + react you can execute the following command to initialize a small example app:
+The two guiding principles are
 
-If you have npm >= 5.2.0 installed:
+1. Hops scales with your requirements, from an easy start up to large scale applications with many teams
+1. Hops fits a broad need by being modular and extensible, it comes with reasonable defaults, but allows you to customize, configure, and extend almost everything when needed
```

Comment:
How about "Hops comes with reasonable defaults, but allows you to customize, configure, and extend almost everything"?

Reply:
I thought it would be most important to point out that hops serves a broad variety of needs. At least that is very important for me.